app/views/layouts/application.html.erb

```diff
@@ -13,20 +13,20 @@
 
 <% content_for :body_end do %>
   <script>
-  var loadedCalendars = [];
+  var loadedModules = [];
   $(document).on('turbolinks:load', function() {
-    var calendars = $('[data-calendar]');
+    var modules = $('[data-module]');
 
-    if (loadedCalendars.length) {
+    if (loadedModules.length) {
```

Comment:
I suppose strictly we should check `loadedModules` for the presence of the specific module we're about to kick off, but as they're all on the same page I don't think this is a big issue for now.